### PR TITLE
AC Automated Test - LRAC-8627 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRAC-8627
https://issues.liferay.com/browse/LRAC-9328 (Refactor)
https://issues.liferay.com/browse/LRAC-9329 (Bug Ticket)

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase [hunk 1]
```diff
@@ -74,12 +74,79 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8627 | Test Summary: Segment distribution chart can be filtered by number property"
-	@ignore = "true"
 	@priority = "3"
 	test CanDistributionChartFilteredNumber {
+		property portal.upstream = "quarantine";
+
+		// AC Refactor: Chart fails to appear when filtering with a number property.
+		// AC Refactor ticket: LRAC-9328
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:82]: Hi @epark18, I think your problem is related to this bug [LRAC-9329](https://issues.liferay.com/browse/LRAC-9329). Could you comment on your test case with the bug ticket?

Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:82]: Hi @epark18 , you can update the code like in this test https://github.com/liferay-qa/liferay-portal/pull/992/files#diff-ac2930901aef96d73ae723bc085193021d39dfb0a5a2f1be646573a314c7208cR280

Just adding a comment:

`// AC Bug ticket: LRAC-9329`

Thanks!

Comment (epark18) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsDistribution.testcase:82]: Done! Thank you.

**Comment by alexcandidos** [comment]
Hi @epark18 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**